src/main/java/teammates/common/util/AbsoluteUrl.java

```diff
@@ -0,0 +1,22 @@
+package teammates.common.util;
+
+/**
+ * An extension of the {@link Url} class which automatically appends the
+ * application's URL (as defined in {@link Config}) to the front of the URL input.
+ */
+public class AbsoluteUrl extends Url {
+
+    public AbsoluteUrl(String url) {
+        super(Config.APP_URL + url);
+    }
+    
+    /**
+     * A shortcut method to create an absolute URL out of a given input.
+     * Use it with caution as it will not check whether the given URL input
+     * is already absolute or not.
+     */
+    public static String makeAbsoluteUrl(String url) {
+        return Config.APP_URL + url;
```

Comment:
I am okay with the static method... I see that it comes in handy in some places :-) But please ensure that you check if the url is already absolute (just for defensive coding)